docs/designs/011-kubernetes-object-monitor.md

```diff
@@ -281,3 +367,29 @@ resource.reason == 'Failed' &&
 (now - timestamp(resource.eventTime)) < duration('5m')  // Only match recent events
 ```
 When the event becomes older than 5 minutes, the predicate returns false → healthy event sent automatically.
+
+
+### Operational Configuration (Container Args/Env Vars)
+
+Controller-level settings are configured via command-line flags in the Deployment:
+
+| Flag | Type | Default | Description |
+|------|------|---------|-------------|
+| `--resync-period` | duration | `5m` | Periodic reconciliation interval for all watched resources |
+| `--max-concurrent-reconciles` | int | `10` | Maximum number of resources to reconcile concurrently |
+| `--policy-config-path` | string | `/etc/nvsentinel/config/policies.toml` | Path to policy configuration file |
+| `--metrics-bind-address` | string | `:8080` | Address to bind Prometheus metrics endpoint |
+| `--health-probe-bind-address` | string | `:8081` | Address to bind health probe endpoints |
+| `--log-level` | string | `info` | Log level (debug, info, warn, error) |
+
+## Metrics
+
+The Kubernetes Object Monitor exposes Prometheus metrics for observability and debugging.
+
+| Metric Name | Type | Labels | Description |
+|------------|------|--------|-------------|
+| `k8s_object_monitor_policy_matches_total` | Counter | `policy_name`, `node`, `resource_kind` | Total number of times a policy matched (predicate=true), indicating detected issues by node |
+| `k8s_object_monitor_active_unhealthy_conditions` | Gauge | `policy_name`, `node`, `resource_kind` | Current number of resources with active unhealthy conditions |
```

Comment:
I'm not sure if we need this gauge. We can always get this info using kube-state-metrics or from the mongodb

Reply:
I don't have any strong opinion on this but I've found mongodb not query-able from outside of the cluster, and also kube-state-metric won't have the generality and it will be split across multiple metrics.

Reply:
> I don't have any strong opinion on this but I've found mongodb not query-able from outside of the cluster

Yes, but we have a work item in this sprint to change that, we are going to continuously export mongodb to a datalake which will be quarriable.